### DISCUSSION TITLE
Fatal Error: Can't read config file

### DISCUSSION THREAD

**Discussion by Yucked** [0xERR0R/blocky#154] [Q&A] [answered]
When running with Docker, the program exists with the following message:
```
Unable to find image 'spx01/blocky:latest' locally
latest: Pulling from spx01/blocky
b538f80385f9: Pull complete
ad486b4f8f76: Pull complete
6bcb09329cd1: Pull complete
194b01c6b856: Pull complete
642f8ce18195: Pull complete
d6295e0f7cdc: Pull complete
Digest: sha256:1b36173883300b65d8040b0305ef1bdba20f47855ff55d99bdef158572cf7c62
Status: Downloaded newer image for spx01/blocky:latest
time="2020-10-04T02:58:12Z" level=fatal msg="Can't read config file: read /app/config.yml: is a directory"```

I'm unsure if it's related to the app itself or Docker.

**Comment by 0xERR0R** [comment]
Did you create the config.yaml. This file must exist and will mount as volume. See wiki for a complete docker example

**Comment by Yucked** [comment]
Yes. I did 
`wget https://raw.githubusercontent.com/0xERR0R/blocky/master/docs/config.yml`

Then ran `docker run --name blocky -v config.yml:/app/config.yml -p 4000:4000 -p 55555:55555/udp spx01/blocky`

But yielded said result.

**Comment by 0xERR0R** [comment, marked as answer]
You must specify the absolute path for docker volume. Please use this command:
`docker run --rm --name blocky -v $(pwd)/config.yml:/app/config.yml -p 4000:4000 -p 55555:55555/udp spx01/blocky`

Example config.yml specifies "/logs" directory for logs, so please specify another docker volume for this:
`docker run --rm --name blocky -v $(pwd)/config.yml:/app/config.yml -v $(pwd)/logs:/logs -p 4000:4000 -p 55555:55555/udp spx01/blocky`

**Comment by Yucked** [comment]
Thank you!
